.circleci/config.yml

```diff
@@ -57,7 +57,7 @@ aliases:
        export UVCDAT_ANONYMOUS_LOG=False
        # run tests again but with cdms2 installed
        set +e
-       conda install -q -n py$PYTHON_VERSION -c conda-forge cdms2
+       conda install -q -n py$PYTHON_VERSION -c cdat/label/nightly -c conda-forge cdms2
```

Comment:
@mauzey1 I assume there's no problem in using a `cdms nightly` build for these tests?

Reply:
@durack1 We were originally using the nightly build of cdms2 for cmor, but I decided to switch to the conda-forge version since it was using a later version of libnetcdf.  However, I decided to switch back to the nightly version of cdms2 and pin libnetcdf 4.6.2 since that version is pinned for cdms.